.circleci/config.yml

```diff
@@ -61,17 +61,39 @@ jobs:
             python setup.py sdist
             twine check dist/*
 
+      - run:
+          name: Update TemplateFlow skeleton
+          command: |
+            cd $HOME/.cache/templateflow/
+            find . -type d -not -path "*/\.*" -exec mkdir -p /tmp/skel/{} \;
+            for i in $( find . -type f -not -path "*/\.*" ); do cp $i /tmp/skel/${i:2}; done
+            find . -type l -not -path "*/\.*" -exec touch /tmp/skel/{} \;
+            cd /tmp/skel
+            mkdir -p /tmp/resources
+            zip -r /tmp/resources/templateflow-skel.zip */
```

Comment:
What's the point of this?

Reply:
To update the skeleton for release. Instead of doing it only when releasing, I pushed it here so that we can check the file as an artifact.

Reply:
Does that sound reasonable?

Reply:
Yeah, that seems fine.